tests/test_recall.py

```
@@ -0,0 +1,69 @@
+    ids = [node.id for node in m.list_view(repo)]
+    m.write_nap(repo, ids[0], ids[1], "folded pair")
+    monkeypatch.setattr(m, "WAKE_LINES", 1)
+    out = m.recall_text(repo, "folded")
+    assert "folded pair" in out
+
+
+def test_recall_matches_sentence_inside_tree(tmp_path):
+    """Recall finds an original sentence that lives only inside a .tree."""
+    m = load_summem()
+    repo = init_repo(tmp_path / "r")
+    m.write_note(repo, "unique-leaf-sentence", datetime(2026, 1, 1, 0, 0, 1, tzinfo=UTC), Random(1))
```

Ruff (0.16.1)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)